src/cutscenes/reimu_good_end.inc.h

```diff
@@ -0,0 +1,77 @@
+		T_NARRATOR("The instigators, Kurumi and Elly, had been conscripted into ‘community service’ by the Moriya Shrine Gods."),
+		T_NARRATOR("Although the bounds of that ‘community service’ seemed to expand as time went on…"),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/02", {
+		T_KANAKO("I can’t believe it! Is nuclear fusion really that simple?!"),
+		T_ELLY("Y-Yes, it’s just a matter of changing that coefficient there, and shaping the magnetic containment field using-"),
+		T_KANAKO("Of course! I was such a fool! I can’t believe I ever bothered attempting to harness the Yatagarasu!"),
+		T_KANAKO("Finally, I have the final key to modernizing Gensokyo, and preventing the energy crisis!"),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/02", {
+		T_ELLY("O-oh dear… are you sure you’re well, ma’am?"),
+		T_SUWAKO("Heh heh, don’t worry, thisis as normal as it gets for Kanako."),
+		T_ELLY("‘Normal’? Really?"),
+		T_SUWAKO("Kanako is a green energy fanatic. Don’t you think she’s cute when she’s like this?"),
+		{ 0 },
+	}},
+	{ "cutscenes/locations/moriya", {
+		T_NARRATOR("Reimu was spending more time on Yōkai Mountain, claiming she was ‘keeping watch’."),
+		T_NARRATOR("For some reason, she seemed to have a difficult time settling down from this incident in particular."),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/03", {
+		T_REIMU("Hey, Elly."),
+		T_ELLY("Ah, yes, what is it?"),
+		T_REIMU("I’m still surprised you’re not still pissed off at everyone. Not even a little bit."),
+		T_REIMU("You were shouting like a dictator up there."),
+		T_ELLY("In the end, it seems the Tower got to me, too."),
+		T_ELLY("I’m still sad about my own Gensokyo, but to go that far…"),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/03", {
+		T_REIMU("Hey, Elly."),
+		T_ELLY("Ah, yes, what is it?"),
+		T_REIMU("I’m still surprised you’re not still pissed off at everyone. Not even a little bit."),
+		T_REIMU("You were shouting like a dictator up there."),
+		T_ELLY("In the end, it seems the Tower got to me, too."),
+		T_ELLY("I’m still sad about my own Gensokyo, but to go that far…"),
+		{ 0 },
+	}},
+
+	{ "cutscenes/reimu_good/03", {
+		T_REIMU("What a weird machine. I wonder what the kappa think about it."),
+		T_SUWAKO("The kappa won’t touch the thing. Even Kanako has her reservations."),
+		T_REIMU("Still, it’s odd how there were forgotten yōkai from another Gensokyo…"),
+		T_REIMU("Ah, whatever. As long as it stays quiet, that Tower isn’t even the weirdest thing to arrive in this place."),
+		{ 0 },
+	}},
+	{ "cutscenes/locations/moriya", {
+		T_NARRATOR("The Tower of Babel continued to loom over Yōkai Mountain, as life returned to normal."),
+		T_NARRATOR("Even as some grew to accept it, there were frequent mutterings over potentially getting rid of the Tower for good."),
+		T_NARRATOR("Although, nobody had any clue how, except, perhaps, a certain violet-tinged gap yōkai…"),
+		{ 0 },
+	}},
+}
```

Comment:
Careful, you missed the `{ NULL }` terminator entry here (see `reimu_bad_end.inc.h`). The game will read garbage past the array bounds and crash without that, depending on the memory layout and the alignment of the stars. ASan should catch this. The joys of C.